Automattic/delphin
# Checkout: add order summary to checkout form

This adds an Order Summary section in the checkout form:

![image](https://cloud.githubusercontent.com/assets/230230/16080519/85a132d2-3309-11e6-99ac-e62389145214.png)
### Testing Instructions
- `git checkout add/checkout-order-summary`
- Select a site and go to checkout
- Validate that it matches the prototype
- Check that the toggle button actually toggles
### Reviews
